main/src/library/Map.flix

```diff
@@ -236,7 +236,8 @@ mod Map {
                 if (useParallelEvaluation(m))
                     let h = (kl, _, kr, _) -> g(kl, kr);
                     let Map(t) = m;
-                    RedBlackTree.parMinimumBy(threads() - 1, h, t)
+                    let threads = unchecked_cast(Environment.handle(threads)() as _ \ {});
```

Comment:
Same comment about not passing the number of threads here. RedBlackTree can get them by themselves.